packages/ui/components/form/inputs/Input.tsx

```diff
@@ -12,18 +12,23 @@ import { FiEye, FiEyeOff, FiX } from "../../icon";
 import { HintsOrErrors } from "./HintOrErrors";
 import { Label } from "./Label";
 
```

Comment:
Added dark mode styles to input fields. This way I could remove a lot of custom styles from the booker form components 🫶